Source/WebKit/WebProcess/Plugins/PluginView.cpp

```diff
@@ -1241,6 +1241,14 @@ bool PluginView::pluginDelegatesScrollingToMainFrame() const
     return m_plugin->delegatesScrollingToMainFrame();
 }
 
+bool PluginView::presentingLockedContent() const
+{
+    if (!m_isInitialized)
+        return false;
+
+    return m_plugin->isLocked();
```

Comment:
This could also be written as:

```cpp
return m_isInitialized && m_plugin->isLocked();
```

Unless you feel the current version is more readable.

Reply:
This style follows the file convention, but I don't think your suggestion is any less readable, so I'll go with it